src/ui/components/Comments/TranscriptComments/CommentCard.tsx

```diff
@@ -266,7 +279,8 @@ function CommentCard({
     <div
       className={classNames(
         `comment-card relative mx-auto w-full cursor-pointer border-b border-splitter transition`,
-        isOutsideFocusedRegion ? "opacity-30" : "bg-bodyBgcolor"
+        isOutsideFocusedRegion ? "opacity-30" : "bg-bodyBgcolor",
+        isUpdating ? "bg-themeBase-90" : ""
```

Comment:
This is the style that makes the in-flight request _look_ disabled.

Separately, we were already hiding the actions "..." button.

Reply:
Oooh, I wonder if that was b'c the previous change was opacity or because @jonbell-lot23 didn't like the style treatment similarity?

I'll follow up with Jon on this. Thank you!